app/Connect/Commands/SubmitCodeNote.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Connect\Commands;
+
+use App\Models\Game;
+use App\Models\MemoryNote;
+use App\Platform\Services\VirtualGameIdService;
+use Illuminate\Http\Request;
+
+class SubmitCodeNote extends AuthenticatedApiHandlerBase
+{
+    public int $gameId;
+    public int $address;
+    public string $note;
```

Comment:
Commands expose their inputs as public fields so they can be used without instantiating a request (see `update-code-note.php` below).